internal/sink/kafka_connect.go

```diff
@@ -0,0 +1,57 @@
+package sink
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type KafkaConnect struct {
+	config *KafkaConnectConfig
+}
+
+type KafkaConnectConfig struct {
+	Address string
+}
+
+type Connector struct {
+	Name   string            `json:"name"`
+	Config map[string]string `json:"config"`
+}
+
+func NewKafkaConnect(config *KafkaConnectConfig) (*KafkaConnect, error) {
```

Comment:
There is no need for either the config or the KafkaConnect struct to be pointers. Since we don't actually return any errors, maybe skip it (unless you know something I don't).

Also, since the config struct has a single member, maybe just pass that value to the constructor function.